mantidproject/mantid
# PyChop update

Updates the flux look-up table for MAPS. Fixes minor bugs (incorrect FWHM vs energy transfer for multi-rep, incorrect approximate formula for FWHM vs energy transfer).

The original issue also mentions that the flux on MERLIN has changed. However, the measurements of this has just been carried out and the results are not yet processed. It will be added in another PR soon.

**To test:**

Run the PyChop interface and check that the flux looks something like the picture on the right rather than the left. Check that when multirep is selected that the lines are not all horizontal but have a curvature as in the picture.

![pychop_fixes](https://user-images.githubusercontent.com/14106963/36567997-5f805634-1820-11e8-89ec-a1db08ff860c.png)

Fixes #21814

**Release Notes** - Added

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
